### PR TITLE
Use "embed" to store kubelet config template

### PR DESCRIPTION
**Pull request by Fedosin**
This commit starts using "embed" go module to store the template in the binary. It is easier and more natural than doing so with go-bindata.

### CODE REVIEW

**Review by sebsoto — CHANGES_REQUESTED** [review]
Thanks for the PR @Fedosin , I wasn't aware of the embed library.

File: pkg/bootstrapper/bootstrapper.go [hunk 1]
```diff
@@ -105,6 +106,9 @@ var (
 
 	// verbosityRegex searches for the verbosity option given to the kubelet
 	verbosityRegex = regexp.MustCompile(`--v=(\w*)`)
+
+	//go:embed templates/kubelet_config.json
+	f embed.FS
```
[the remark this answers is not in the document]
Comment (Fedosin) [reply pkg/bootstrapper/bootstrapper.go]: Done!

**Review by saifshaikh48 — COMMENTED** [review]
/lgtm

Thanks for working on this @Fedosin.

**Pull request merged**